opae.spec.fedora

```diff
@@ -177,10 +181,36 @@ done
 %{_libdir}/libopaeuio.so.*
 %{_libdir}/libopaevfio.so.*
 
-%post devel
+%{_libdir}/opae/libxfpga.so
+%{_libdir}/opae/libopae-v.so
+%{_libdir}/opae/libmodbmc.so
+%{_libdir}/opae/libfpgad-xfpga.so
+%{_libdir}/opae/libfpgad-vc.so
+%{_libdir}/opae/libboard_a10gx.so
+%{_libdir}/opae/libboard_n3000.so
+%{_libdir}/opae/libboard_d5005.so
+%{_libdir}/opae/libboard_n5010.so
+%{_libdir}/opae/libboard_n6000.so
+
+%{_bindir}/fpgad
+%{_bindir}/fpgaconf
+%{_bindir}/fpgainfo
+%{_bindir}/fpgasupdate
+%{_bindir}/rsu
+%{_bindir}/pci_device
+
+%{python3_sitelib}/opae*
```

Comment:
it would be nice to group like paths together...
maybe put this lower, around line 305ish?